javaharness/java/arcs/android/AndroidRuntimeSettings.java

```diff
@@ -32,13 +32,18 @@
   // Does *not* connect Arcs Explorer
   private static final boolean DEFAULT_ENABLE_ARCS_EXPLORER = false;
   // Loads the on-device pipes-shell
-  private static final String DEFAULT_SHELL_URL = "file:///android_asset/arcs/index.html?";
+  // Multiple protocols are supported, i.e.
+  // file:///android_asset/arcs/index.html?
+  // https://appassets.androidplatform.net/assets/arcs/index.html?
+  // The Arcs Cache Manager only works at https secure origin.
+  private static final String DEFAULT_SHELL_URL =
+      "https://appassets.androidplatform.net/assets/arcs/index.html?";
   // Load the on-device assets
   private static final boolean DEFAULT_ASSETS_FROM_WORKSTATION = false;
   // Uses the standard 8786 port
   private static final int DEFAULT_DEV_SERVER_PORT = 8786;
-  // Deactivates the Arcs Cache Manager.
-  private static final boolean DEFAULT_USE_CACHE_MANAGER = false;
+  // Activates the Arcs Cache Manager.
+  private static final boolean DEFAULT_USE_CACHE_MANAGER = true;
```

Comment:
If we need to turn this off again for some reason, will a subsequent PR to do that just need to flip this to `false`, or do we need to do some of the other edits in this PR (such as the one in `cache-mgr-sw.js` below)?

Reply:
To turn off this feature temporarily, using the single adb command:
> adb shell "setprop debug.arcs.runtime.use_cache_mgr false"
The hardcoded DEFAULT_USE_CACHE_MANAGER will be overridden by this command.

To rollback to file:/// protocol, using the single adb command:
> adb shell "setprop debug.arcs.runtime.shell_url 'file:///android_asset/arcs/index.html?'"

Combine these two commands will roll everything dynamically back to that before the change.
It's convenient for devs to verify w/ vs. w/o the feature.

Reply:
To turn off this feature for good, yes, flip the flag to false in new PR is good enough (no service worker is created at run-time anymore).